pkg/dockerbuild/spec_test.go

```diff
@@ -44,28 +46,30 @@ func TestBuild_Node(t *testing.T) {
 
 	opts := append([]cmp.Option{cmpopts.EquateEmpty()}, option.CmpOpts()...)
 	c.Assert(spec, qt.CmpEquals(opts...), &ImageSpec{
-		Entrypoint: []string{"/artifacts/0/build/entrypoint"},
+		Entrypoint: []string{"/image/artifacts/entrypoint"},
 		Env: []string{
 			"ENCORE_RUNTIME_LIB=/encore/runtimes/js/encore-runtime.node",
 		},
 		WorkingDir: "/",
 		BuildInfo:  BuildInfoSpec{InfoPath: defaultBuildInfoPath},
 		WriteFiles: map[ImagePath][]byte{defaultMetaPath: meta},
 		CopyData: map[ImagePath]HostPath{
-			"/artifacts/0/build":        "/host/artifacts",
-			"/artifacts/0/package.json": "/host/package.json",
-			"/artifacts/0/node_modules": "/host/node_modules",
-			"/encore/runtimes/js":       "/host/runtimes/js",
+			"/encore/runtimes/js": "/host/runtimes/js",
 		},
-		BundleSource:    option.Option[BundleSourceSpec]{},
+		BundleSource: option.Some(BundleSourceSpec{
+			Source:         "/host",
+			Dest:           "/image",
+			AppRootRelpath: ".",
+			ExcludeSource:  []RelPath{},
+			IncludeSource:  []RelPath{},
+		}),
 		Supervisor:      option.None[SupervisorSpec](),
 		BundledServices: []string{"bar", "foo"},
 		BundledGateways: []string{"baz", "qux"},
 		DockerBaseImage: "scratch",
 		FeatureFlags:    map[FeatureFlag]bool{NewRuntimeConfig: true},
 		StargzPrioritizedFiles: []ImagePath{
-			"/artifacts/0/package.json",
-			"/artifacts/0/build/entrypoint",
+			"/image/artifacts/entrypoint",
```

Comment:
Are we still producing this `artifacts` directory? I believe it's not really needed anymore since the entrypoint is under `/workspace`?

Reply:
no, this is just from what we set the entrypoint to in this test, the /artifacts/0/... are removed from typescript images